Assignment2/felipeleonfernandez-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,60 @@
+{
+	"@context": {
+		"vocab": "http://devices.com/#",
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"Class01": "vocab:Class01",
+		"Sensor029": "Class01:Sensor029",
+		"Computer101": "Class01:Computer101",
+		"Measurement8401": "Sensor029:Measurement8401",
+		"User10A": "Computer101:User10A",
+		"includes": "rdfs:label",
+		"hasMeasurement": "rdfs:label",
+		"hasTemperature": "rdfs:label",
+		"atTime":"rdfs:label",
+		"hasOwner": "rdfs:label",
+		"hasName": "rdfs:label"
```

Comment:
Syntactically is ok, but you have to define each attribute with a specific vocabulary attribute name because when you want to retrieve, for example your atTime, if you query "rdfs:label" you will not be able to distinguish what data you ask for. If you have your own vocabulary you can use it to define your attributes e.g. "atTime":"vocab:atTime", "includes":"vocab:includes", etc.